### ISSUE TITLE
error 

### ISSUE THREAD

**Issue by buddhix** [mfrashad/text2art#4]
ImportError                               Traceback (most recent call last)
<ipython-input-3-25d790141602> in <module>()
----> 1 import clipit
      2 # To reset settings to default
      3 clipit.reset_settings()
      4 # You can use "|" to separate multiple prompts
      5 prompts = "underwater city"

10 frames
/usr/local/lib/python3.7/dist-packages/torch_xla/__init__.py in <module>()
    126 import torch
    127 from ._patched_functions import _apply_patches
--> 128 import _XLAC
    129 
    130 

ImportError: /usr/local/lib/python3.7/dist-packages/_XLAC.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _ZN2at11result_typeERKNS_6TensorEN3c106ScalarE

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.

**Comment by szymekdr** [comment]
Similar here:

```
ImportError                               Traceback (most recent call last)
<ipython-input-5-25d790141602> in <module>()
----> 1 import clipit
      2 # To reset settings to default
      3 clipit.reset_settings()
      4 # You can use "|" to separate multiple prompts
      5 prompts = "underwater city"

12 frames
/usr/local/lib/python3.7/dist-packages/torchtext/vocab/vocab_factory.py in <module>()
      2 from typing import Dict, Iterable, Optional, List
      3 from collections import Counter, OrderedDict
----> 4 from torchtext._torchtext import (
      5     Vocab as VocabPybind,
      6 )

ImportError: /usr/local/lib/python3.7/dist-packages/torchtext/_torchtext.so: undefined symbol: _ZTVN5torch3jit6MethodE
```

**Comment by CaptivatingCatz** [comment]
bump issue confirmed

**Comment by mfrashad** [comment]
I've updated the repo. Fixed with `pip install torchtext==0.10.0`.
